src/me/mrCookieSlime/Slimefun/Setup/ResearchSetup.java

```diff
@@ -105,7 +105,7 @@ public static void setupResearches() {
 	    Slimefun.registerResearch(new Research(98, "Advanced Mining 101", 42), SlimefunItems.ADVANCED_DIGITAL_MINER);
 	    Slimefun.registerResearch(new Research(99, "Composting Dirt", 3), SlimefunItems.COMPOSTER);
 	    Slimefun.registerResearch(new Research(100, "Farmer Shoes", 4), SlimefunItems.FARMER_SHOES);
-	    Slimefun.registerResearch(new Research(101, "Explosive Pickaxe", 28), SlimefunItems.EXPLOSIVE_PICKAXE);
+	    Slimefun.registerResearch(new Research(101, "Maximum Explosivity", 30), SlimefunItems.EXPLOSIVE_PICKAXE, SlimefunItems.EXPLOSIVE_SHOVEL);
```

Comment:
Rather just call it "Explosive Tools", this sounds a bit exaggerated.

Reply:
Alright I'll fix it now and also the placeholder method in SlimefunSetup